diff --git a/kernel/Makefile b/kernel/Makefile
index c48693d1..648bb24c 100644
--- a/kernel/Makefile
+++ b/kernel/Makefile
@@ -12,7 +12,7 @@ $(info -- KDIR: $(KDIR))
 $(info -- MDIR: $(MDIR))
 
 # KSU VERSION
-KSU_VERSION := 33129
+KSU_VERSION := 33300
 KSU_VERSION_TAG := v3.1.0
 
 ccflags-y += -DKSU_VERSION=$(KSU_VERSION)
diff --git a/kernel/hook/core_hook.c b/kernel/hook/core_hook.c
index bbd1090a..e21ae681 100644
--- a/kernel/hook/core_hook.c
+++ b/kernel/hook/core_hook.c
@@ -61,9 +61,6 @@ static void ksu_handle_extra_susfs_work(void)
         pr_err("susfs: Failed adding task_work 'susfs_handle_setuid_tw', err: %d\n", err);
     }
 }
-#ifdef CONFIG_KSU_SUSFS_TRY_UMOUNT
-extern void susfs_try_umount(uid_t uid);
-#endif // #ifdef CONFIG_KSU_SUSFS_TRY_UMOUNT
 #endif // #ifdef CONFIG_KSU_SUSFS
 
 LSM_HANDLER_TYPE ksu_handle_rename(struct dentry *old_dentry, struct dentry *new_dentry)
@@ -151,11 +148,7 @@ do_umount:
 #endif // #ifdef CONFIG_KSU_SUSFS_SUS_MOUNT
 
     // Handle kernel umount
-#ifndef CONFIG_KSU_SUSFS_TRY_UMOUNT
     return ksu_handle_umount(new, old);
-#else
-    susfs_try_umount(new_uid);
-#endif // #ifndef CONFIG_KSU_SUSFS_TRY_UMOUNT
 
 #ifdef CONFIG_KSU_SUSFS_SUS_PATH
     //susfs_run_sus_path_loop(new_uid);
diff --git a/kernel/supercall/supercall.c b/kernel/supercall/supercall.c
index 8469d48c..2401a2b1 100644
--- a/kernel/supercall/supercall.c
+++ b/kernel/supercall/supercall.c
@@ -149,12 +149,6 @@ int ksu_handle_sys_reboot(int magic1, int magic2, unsigned int cmd, void __user
             return 0;
         }
 #endif //#ifdef CONFIG_KSU_SUSFS_SUS_KSTAT
-#ifdef CONFIG_KSU_SUSFS_TRY_UMOUNT
-        if (cmd == CMD_SUSFS_ADD_TRY_UMOUNT) {
-            susfs_add_try_umount(arg);
-            return 0;
-        }
-#endif //#ifdef CONFIG_KSU_SUSFS_TRY_UMOUNT
 #ifdef CONFIG_KSU_SUSFS_SPOOF_UNAME
         if (cmd == CMD_SUSFS_SET_UNAME) {
             susfs_set_uname(arg);
